src/view/index.js

```diff
@@ -1,2 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Internal dependencies
+ */
 import './style.scss';
 import './patterns.scss';
+
+const manageForms = container => {
+	const forms = Array.from( container.querySelectorAll( 'form.popup-action-form' ) );
```

Comment:
I haven't verified but in these requests the Mailchimp block would use the non-AMP implementation, which already does an XHR request, wouldn't it?

Reply:
Yes, the Mailchimp block is covered. But if the user places a [Form block](https://wordpress.com/support/wordpress-editor/blocks/form-block/) in the Campaign, then it will be submitted like a traditional form. 
That's maybe not a very common case, but a Donate block is – and it renders a regular `form` element, right?

Reply:
Right, but in both of those cases wouldn't we _want_ them to behave the way they would normally in a non-AMP context? The main motivation behind this work is to get Forms working normally again, without interference from `amp-form`.

Reply:
I think it's fine, but AMP provides a better UX by submitting all "non-AMP" `method=GET` forms asynchronously. Just wanted to make sure that we're on the same page.